setup.cfg

```diff
@@ -50,7 +50,7 @@ install_requires =
     pycryptodomex  # for EncryptedKeyring backend in keyrings.alt
     pydantic >= 1.9.0
     pynwb >= 1.0.3,!=1.1.0
-    nwbinspector >= 0.3.8
+    nwbinspector >= 0.4.12
```

Comment:
Figured I might as well bump this while I'm at it - the `get_package_version` convenience helper is only available in `>=0.4.10`.

Point still being this isn't something that would need to be done hereafter since the version check in the validation will require the latest update.

Reply:
ok